.github/dependabot.yaml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
+  - package-ecosystem: "npm"
```

Comment:
yes, please revert

Reply:
So... I don't understand why this file because  dependency-review for test suite is [there](https://github.com/MichMich/MagicMirror/blob/master/.github/workflows/depsreview.yaml)

I think create an dependabot monthly is not bad.
sure... we have to check all dependency found but it can help
 * a little time saving
 * just checking is dependency is on package.json (for not update an sub-dependency)
 * able to test the dependabot branch

in all case i will reverse it